api/v1/helpers/verbs/utils.js

```diff
@@ -87,8 +93,21 @@ function handleUpdatePromise(resultObj, req, retVal, props, res) {
     redisCache.del(findCacheKey);
   }
 
+  resultObj.dbTime = new Date() - resultObj.reqStartTime;
+  logAPI(req, resultObj, returnObj);
+
   return res.status(constants.httpStatus.OK)
-    .json(responsify(retVal, props, req.method));
+    .json(responsify(returnObj, props, req.method));
+}
+
+/**
+ * Sorts an array in-place.
+ *
+ * @param {Array} arr Array of objects
+ * @param {String} fieldName The field to sort by
+ */
+function sortArrayObjectsByField(arr, fieldName) {
+  arr.sort((a, b) => a[fieldName].localeCompare(b[fieldName]));
```

Comment:
arr.sort((a, b) => a[fieldName].localeCompare(b[fieldName])); is the only thing this function is doing.  Probably does not add value to make it a function ?

Reply:
making it into a function is good for:
1) testing
2) function header describes what the one liner does.
Any other thoughts or comments or feedback?